connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTest.java

```diff
@@ -90,6 +92,33 @@
     private WorkerConfig config;
     private Worker worker;
 
+    private final Map<String, String> defaultProducerConfigs = defaultProducerConfigs();
+
+    private Map<String, String> defaultProducerConfigs() {
```

Comment:
nit: Seems weird that we make these methods, but then use a `private final` that just invokes this everywhere. Just initialize in a `static` block or call `defaultProducerConfigs()` directly in each test?